package.json

```diff
@@ -7,6 +7,7 @@
   "bugs": "https://github.com/hypothesis/client/issues",
   "repository": "hypothesis/client",
   "devDependencies": {
+    "@octokit/rest": "^15.13.0",
```

Comment:
This is a REST API client for the [GitHub API v3](https://developer.github.com/v3/). The current API (v4) is GraphQL-based. This one is not going away in a hurry and is fine for our needs though.

Reply:
_yuge_, I laughed. I sure do hope that GH API v3 sticks around, as it is my preferred go-to model for "REST done right on the Web"!

Reply:
I reckon hundreds of thousands of lines of existing code in open source and enterprise projects will keep it around for a while, much to the chagrin of GitHub devs no doubt 😁.

I can imagine why GraphQL might be a better fit for GitHub as there are lots of linked resources that a particular client application might want to traverse in a single task.  I've not actually tried using the v4 API so I can't say much about it.